docs/runbooks/search-functionality/resident-search.md

```diff
@@ -0,0 +1,96 @@
+---
+id: resident-search
+title: Resident Search Configuration
+slug: /runbooks/search-functionality/resident-search
```

Comment:
This isn't in the correct runbook, it should be under `/runbooks/case-viewer` as this functionality is part of that application.

Reply:
Thanks, I'll move that